library/ibm-semeru-runtimes

```diff
@@ -0,0 +1,92 @@
+# IBM Semeru Runtimes official Docker images.
+
+Maintainers: Surya Narkedimilli <snarkedi@in.ibm.com> (@narkedi)
+GitRepo: https://github.com/ibmruntimes/semeru-containers.git
+
+#-----------------------------openj9 v8 images---------------------------------
+Tags: open-8u302-b08-jdk-focal, open-8-jdk-focal
+SharedTags: open-8u302-b08-jdk, open-8-jdk
+Architectures: amd64, ppc64le, s390x
+GitCommit: 21d1d498b7dff9b58acd3c265f1bb3aca97ce23e
+Directory: 8/jdk/ubuntu
+File: Dockerfile.open.releases.full
+
+Tags: open-8u302-b08-jdk-centos7, open-8-jdk-centos7
+SharedTags: open-8u302-b08-jdk, open-8-jdk
```

Comment:
Unfortunately `SharedTags` does not work for items of the same [`Platform`](https://specs.opencontainers.org/image-spec/image-index/?v=v1.0.1#:~:text=be%20safely%20ignored.-,platform%20object,-This%20OPTIONAL%20property) and so would create an invalid manifest when used for two Linux-based images or two Windows images with the same `os.version` (e.g. `nanoserver-1809` and `windowsservercore-1809`).

So, `SharedTags` is only used for differing platforms: up to one linux (e.g. `ubuntu`), and each supported Windows version (e.g. `windowsservercore-1809` and `windowsservercore-ltsc2022`).

See the shared tags used in the [`golang` library file](https://github.com/docker-library/official-images/blob/5d9206dd7cc736c41fd0f9eedb324e1fd00309e2/library/golang#L8-L67) as a fairly comprehensive example:
 - shared "generic" tags (like `1.17.1`) on `bullseye`, `windowsservercore-ltsc2022`, `windowsservercore-1809`, and `windowsservercore-ltsc2016`
     - but not any of the other Linux distributions (`buster`, `stretch`, `alpine*`) or `nanoserver`
 - shared `windowsservercore`-only tags for  `windowsservercore-{ltsc2022,1809,ltsc2016}`
 - shared `nanoserver`-only tags for `nanoserver-{ltsc2022,1809}`

Reply:
@yosifkit I have stopped using the SharedTags for CentOS. Thanks for referring to the example. !! Will handle it accordingly when we add support for Windows later. Thanks !!